File: src/logic/recReq/recreq.js
// scripts that checks for new files being received. 

// > Creating temporary folders.
// > Deleteing them after userinput on website? 



// Imports


function checkNewRequest() {
    // idk yet 
    
    pass
    
}

function cretaeTempFolder(req_number) {
    x = os.path.exists("data/req/{req_number}/")

    if (x == true) {
        pass
    }
    if (x != true) {
        os.mkdir("data/req/{req_number}/")
    }
    

}

function deleteTempFolder(req_number) {
    x = os.path.exists("data/req/{req_number}/")

    if (x == true) {
        os.rmdir("data/req/{req_number}/")
    }
    if (x != true) {
        pass
    }
    

}

File: src/logic/recReq/recreq.py
"""
scripts that checks for new files being received. 

> Creating temporary folders.
> Deleteing them after userinput on website? 

"""


"""
Imports
"""
import os

def checkNewRequest():
    # idk yet 
    
    pass


def cretaeTempFolder(req_number):
    x = os.path.exists(f"data/req/{req_number}/")

    if x == True:
        pass
    if x != True:
        os.mkdir(f"data/req/{req_number}/")
    


def deleteTempFolder(req_number):
    x = os.path.exists(f"data/req/{req_number}/")

    if x == True:
        os.rmdir(f"data/req/{req_number}/")
    if x != True:
        pass



